libcaja-private/caja-module.c

```diff
@@ -263,6 +263,18 @@ caja_module_setup (void)
     {
         initialized = TRUE;
 
+        const gchar *caja_extension_dirs = g_getenv ("CAJA_EXTENSION_DIRS");
```

Comment:
It's me again :)  I think the style is C89, so the variable declaration should be at the *start* of the scope (e.g. before the `initialized = TRUE;` line.

Reply:
Done.